packages/ddp-server/livedata_server.js

```diff
@@ -988,7 +988,7 @@ Object.assign(Session.prototype, {
     else
       self._universalSubs.push(sub);
 
-    sub._runHandler();
+    return sub._runHandler();
```

Comment:
why return this? Should just await here work?

Reply:
This way you can handle any error where you're calling this method.